### ISSUE TITLE
run yarn build react/index,react-dom/index --type=UMD but noting

### ISSUE THREAD

**Issue by zhang14725804** [reactjs/reactjs.org#2815]
download react-16.8.0 source code and run "yarn"

follow Development Workflow

run "yarn build react/index,react-dom/index --type=UMD"

but react-16.8.0/build/node_modules/react/umd/react.development.js net::ERR_FILE_NOT_FOUND

build folder only contains facebook-www folder and react-native folder

**Comment by threepointone** [comment]
Worked fine for me, and the rest of the team. what version of node are you using? can you share more of the error from your terminal?

**Comment by zhang14725804** [comment]
node version:12.16.0，platform win10,，react 16.8.0

**Comment by threepointone** [comment]
Try with node v10?

**Comment by threepointone** [comment]
Please share what the terminal output is after you build as well

**Comment by zhang14725804** [comment]
![image](https://user-images.githubusercontent.com/15354910/76272318-14c13e00-62b6-11ea-8981-272e358ea097.png)

**Comment by threepointone** [comment]
Yeah it doesn’t seem to be building properly here. Please try using node v10, run yarn once, and try again. We’ll work on making this work with 12 soon.

**Comment by zhang14725804** [comment]
I use node 10.8.0 It work for me! thank you very much! but what the error mean in the follow picture
![image](https://user-images.githubusercontent.com/15354910/76274208-f01b9500-62ba-11ea-9bf2-c532fd1aef05.png)

**Comment by Glenn8119** [comment]
the same issue, also using windows10
I tried node  v14.20.0, v12.22.12 and v10.24.1 and still the same for 'main' and '17.0.2' branch

**Comment by NikbakhtMehdi** [comment]
[(https://react.dev/blog/2024/04/25/react-19-upgrade-guide#umd-builds-removed)]

UMD was widely used in the past as a convenient way to load React without a build step. Now, there are modern alternatives for loading modules as scripts in HTML documents. Starting with React 19, React will no longer produce UMD builds to reduce the complexity of its testing and release process.
